### ISSUE TITLE
Error running new `project-config/rebuild` command

### ISSUE THREAD

**Issue by benface** [craftcms/cms#4062]
### Description

I'm trying to use the new `project-config/rebuild` CLI command and getting this error:

> Rebuilding the project config from the current state ... error: Argument 1 passed to craft\helpers\Json::decodeIfJson() must be of the type string, null given, called in .../vendor/craftcms/cms/src/services/ProjectConfig.php on line 1607

### Additional info

- Craft version: 3.1.20
- PHP version: PHP 7.1.23
- Database driver & version: MySQL 5.7.25

**Comment by brandonkelly** [comment]
Fixed for the next release. 

To get the fix early, change your `craftcms/cms` requirement in composer.json to:

```json
"require": {
  "craftcms/cms": "dev-develop#5713b9ec4865fb058ab46ac4c062e152cd737e99 as 3.1.20",
  "...": "..."
}
```

Then run `composer update`.

**Comment by brandonkelly** [comment]
Just released 3.1.20.1 with this fix.
